process-engine/process_engine_meta
# [WIP] Feature/update major for data model dependencies

Closes #72 
Changes from this PR are included in this feature (that goes for all corresponding packages aswell).

## What did you change?

I've made major bumps on most repositories based on the changes introduced in https://github.com/essential-projects/data_model/pull/12 and https://github.com/essential-projects/data_model_contracts/pull/12

## How can others test the changes?

Execute the whole dev-setup using the `setup.sh` script found in this feature branch.

The integration tests should still succeed, the skeleton projects should still run and the BPMN-Studio should still work.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
